### PR TITLE
[FIX] delete_record_translations: error when only update for 1 one column

### PR DESCRIPTION
**Pull request by duong77476**

### CODE REVIEW

**Review by legalsylvain — APPROVED** [review]

**Comment by MiquelRForgeFlow** [comment]
Please, don't duplicate code. Try to do it in only one query. You can calculate columns and values before the query, and then add `()` in case > 1.

**Comment by duong77476** [comment]
> Please, don't duplicate code. Try to do it in only one query. You can calculate columns and values before the query, and then add `()` in case > 1.

Yes, i understand, i'll will update and rerun again

File: openupgradelib/openupgrade.py [hunk 1]
```diff
@@ -2680,18 +2680,27 @@ def delete_record_translations(cr, module, xml_ids, field_list=None):
...
+                    "jsonb_build_object('en_US', {x} -> 'en_US')".format(x=x)
+                    for x in list_columns
+                )
+                columns = "({})".format(columns)
+                values = "({})".format(values)
+            else:
+                columns = list_columns[0]
+                values = "jsonb_build_object('en_US', {x} -> 'en_US')".format(
+                    x=list_columns[0]
+                )
```
Comment (MiquelRForgeFlow) [inline openupgradelib/openupgrade.py:2697]:
```suggestion
            columns = ", ".join(list_columns)
            values = ", ".join(
                "jsonb_build_object('en_US', {x} -> 'en_US')".format(x=x)
                for x in list_columns
            )
            if len(list_columns) > 1:
                columns = "({})".format(columns)
                values = "({})".format(values)
```

Comment (duong77476) [reply openupgradelib/openupgrade.py]: Thank you, but i prefer keeping my comment is that ok?

Comment (duong77476) [reply openupgradelib/openupgrade.py]: Got it, done sir

**Review by MiquelRForgeFlow — APPROVED** [review]

**Comment by MiquelRForgeFlow** [comment]
Thanks @duong77476

**Pull request merged**